### PR TITLE
Redirect users that fail to connect payment back to store

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Expose `onError` event handler from trustly mutation.

Redirect users that fail to complete the trustly mutation to "hedvig-com".

Skip the `SessionTokenGuard` on the payment page since it doesn't do what we want it to do.

## Why?

We don't want users to get redirected to "/new-member" when they fail to complete the trustly mutation.

**Ticket(s): []**

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1241** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1241" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1241?utm_source=stack-comment).

File: src/client/pages/Payment/Payment.tsx [hunk 1]
```diff
@@ -23,7 +22,13 @@ export const PaymentPage = () => {
   const textKeys = useTextKeys()
   const { path: pathLocale } = useCurrentLocale()
   const redirectURL = useStoredRedirectURL()
-  const [, trustlyURL] = useCreateTrustlyURL()
+
+  const [, trustlyURL] = useCreateTrustlyURL({
+    onError: () => {
```
Comment (alebedev) [inline src/client/pages/Payment/Payment.tsx:27]: Should we add some logs for errors as well?

**Review by alebedev — APPROVED** [review]
Looks good, let's hope it helps with our redirect problems

File: src/client/pages/Payment/Payment.tsx (hunk 1 above)
Comment (robinandeer) [reply src/client/pages/Payment/Payment.tsx:27]: taken care of inside the hook

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: 2UrnAmXoFXyoSIFuHtfn.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1241)

**Pull request merged**